Minimum version of tonic dependency is too low

### What crate(s) in this repo are involved in the problem?

console-api

### What is the issue?

FYI, version 0.8.1 of console-api uses the `tonic::Status::GRPC_STATUS` constant, which was introduced in tonic v0.12.3, but its dependency on tonic only specifies 0.12.

### How can the bug be reproduced?

Update `console-api` or `console-subscriber` in a project with an old Cargo.lock.

### Logs, error output, etc

```
    Checking console-api v0.8.1
error[E0599]: no associated item named `GRPC_STATUS` found for struct `Status` in the current scope
   --> /home/david/.cargo/registry/src/index.crates.io-6f17d22bba15001f/console-api-0.8.1/src/generated/rs.tokio.console.instrument.rs:577:48
    |
577 | ...                   tonic::Status::GRPC_STATUS,
    |                                      ^^^^^^^^^^^ associated item not found in `Status`

error[E0599]: no associated item named `GRPC_STATUS` found for struct `Status` in the current scope
   --> /home/david/.cargo/registry/src/index.crates.io-6f17d22bba15001f/console-api-0.8.1/src/generated/rs.tokio.console.trace.rs:357:48
    |
357 | ...                   tonic::Status::GRPC_STATUS,
    |                                      ^^^^^^^^^^^ associated item not found in `Status`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `console-api` (lib) due to 2 previous errors
```

### Versions

console-api v0.8.1

### Possible solution

Change the dependency specification to `version = "0.12.3"`.

### Additional context

_No response_

### Would you like to work on fixing this bug?

maybe

--- a/console-api/Cargo.toml
+++ b/console-api/Cargo.toml
@@ -29,7 +29,7 @@ keywords = [
 transport = ["tonic/transport"]
 
 [dependencies]
-tonic = { version = "0.12", default-features = false, features = [
+tonic = { version = "0.12.3", default-features = false, features = [
     "prost",
     "codegen",
     "transport",

